pkgs/games/cockatrice/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, kde4, cmake, protobuf, zlib, libgcrypt, fetchurl }:
+
+assert stdenv.isLinux;
+
+stdenv.mkDerivation rec {
+  name = "cockatrice-git-b1736c75";
+  src = fetchurl {
```

Comment:
Why? I have tested cockatrice to work with this particular snapshot of the code; we don't even need git to build this derivation, I don't plan to update this package unless some major incompatibility occurs.